settings/testfiles/airflow_settings_export.yaml

```diff
@@ -1,23 +1,23 @@
 airflow:
   connections:
-    - conn_id: local_postgres
-      conn_type: postgres
-      conn_host: example.db.example.com
-      conn_schema: schema
-      conn_login: username
-      conn_password: password
-      conn_port: 5432
-      conn_uri: postgres://username:password@example.db.example.com:5432/schema
-      conn_extra: {}
+  - conn_id: local_postgres
```

Comment:
nitpick: could we please fix the tests that cause the update to these test files every time we run the tests? Are your changes in this PR to `settings/settings_test.go` fixing them?

Reply:
yes if the yaml file is like this than the tests shouldn't update the file any more